dist/js/tool.js

```diff
@@ -1 +1,1397 @@
\ No newline at end of file
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
Looks like you've committed the dev build of tool bundle. Could you run the prod build and commit that instead?

I will get Actions set up to do this automatically at some point, but don't have the time to do so just now

Reply:
Of course, you're absolutely right! 
I've pushed a production build to this PR. 👍